build-locally.py

```diff
@@ -104,10 +119,10 @@ def main(args=None):
             run_docker_build(ns)
         elif ns.config.startswith("osx"):
             run_osx_build(ns)
+        elif ns.config.startswith("win"):
+            run_win_build(ns)
```

Comment:
We don't support win build, we can just throw and remove the function here

Reply:
this is autogenerated